backend/azure-functions/src/IsSupervisor/index.ts

```diff
@@ -0,0 +1,27 @@
+import { PassedAuthenticationResult, authenticate } from '../Auth/facebook-auth';
+import { Context, HttpRequest } from 'azure-functions-ts-essentials';
+
+const allowedUserIds: string[] = [
+  // TODO: add users
```

Comment:
Mine is 1044662142669968
(Consider leaving this comment opened until having also Shelly and others, so we don't forget this).